web-docs/docs/02-CHANGELOG.mdx

```diff
@@ -26,7 +37,8 @@ toc_max_heading_level: 2
 ### Bug Fixes
 - N/A
 ### Other
-- Migrated temporary abstract package to SC extensions library. There are no functional changes to the connector.
+- Migrated temporary abstract package to SC extensions library. There are no functional changes to
```

Comment:
This change should not be in the release commit

Reply:
Moved to its own commit